### PR TITLE
Fix npe when hessian deserialize

### PR DESCRIPTION
**Pull request by Daydreamer-ia**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change
#10657
XXXXX

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

**Comment by KomachiSion** [comment]
修复方式应该不正确， byte数组如果非法或者是空数组的情况下，应该还是会导致结果是null

**Comment by Daydreamer-ia** [comment]
> byte数组如果非法或者是空数组的情况下

`ByteUtils.isEmpty(data)` 中判断了byte数组是否空以及长度是否为0的逻辑，"byte数组如果非法"是什么意思呢？

File: consistency/src/main/java/com/alibaba/nacos/consistency/serialize/HessianSerializer.java [hunk 1]
```diff
@@ -50,7 +50,13 @@ public <T> T deserialize(byte[] data) {
     
     @Override
     public <T> T deserialize(byte[] data, Class<T> cls) {
+        if (ByteUtils.isEmpty(data)) {
```
Comment (KomachiSion) [inline consistency/src/main/java/com/alibaba/nacos/consistency/serialize/HessianSerializer.java:53]: deseiralize0 方法中已经判断了这个， 没必要判断两次

**Review by KomachiSion — CHANGES_REQUESTED** [review]

File: consistency/src/main/java/com/alibaba/nacos/consistency/serialize/HessianSerializer.java (hunk 1 above)
Comment (Daydreamer-ia) [reply consistency/src/main/java/com/alibaba/nacos/consistency/serialize/HessianSerializer.java]: Fix

**Review by KomachiSion — APPROVED** [review]

**Pull request merged**